api/src/main/java/org/pmiops/workbench/db/model/DataSetValues.java

```diff
@@ -0,0 +1,41 @@
+package org.pmiops.workbench.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.Table;
+
+@Embeddable
+@Table(name = "data_set_cohort_id")
+public class DataSetValues {
+
+  private String domainId;
+  private String value;
+
+  public DataSetValues() {
+  }
+
+  public DataSetValues(String domainId, String value) {
+    this.domainId = domainId;
+    this.value = value;
+  }
+
+
+  @Column(name = "domain_id")
+  public String getDomainId() {
```

Comment:
If possible, it would be good to have this return the Domain itself, or make a helper method titled `getDomainEnum` and `setDomainEnum` to get and set via enum. (See the concept set model for an example)